README.md

```diff
@@ -27,26 +27,37 @@ Sigstore already has a canonical Go client implementation, [cosign](https://gith
 
 You can use the CLI with `go run` as in the below examples, or compile/install the `sigstore-go` CLI:
 
-```bash
+```shell
 $ make install
 ```
 ## Examples
 
-```bash
-$ go run cmd/sigstore-go/main.go -trustedrootJSONpath examples/trusted-root-public-good.json examples/bundle-provenance.json
+```shell
+$ go run cmd/sigstore-go/main.go \
+  -artifact-digest 76176ffa33808b54602c7c35de5c6e9a4deb96066dba6533f50ac234f4f1f4c6b3527515dc17c06fbe2860030f410eee69ea20079bd3a2c6f3dcf3b329b10751 \
+  -artifact-digest-algorithm sha512 \
+  -expectedIssuer https://token.actions.githubusercontent.com \
+  -expectedSAN https://github.com/sigstore/sigstore-js/.github/workflows/release.yml@refs/heads/main \
+  examples/bundle-provenance.json
 Verification successful!
+{
+   "version": 20230823,
+   "statement": {
+      "_type": "https://in-toto.io/Statement/v0.1",
+      "predicateType": "https://slsa.dev/provenance/v0.2",
+      "subject": ...
+    },
+    ...
+}
 ```
 
-```bash
-$ go run cmd/sigstore-go/main.go -tufRootURL tuf-repo-cdn.sigstore.dev examples/bundle-provenance.json
-Verification successful!
-```
+You can also specify a TUF root with something like `-tufRootURL tuf-repo-cdn.sigstore.dev`.
 
 Alternatively, you can install a binary of the CLI like so:
```

Comment:
Is this binary meant for typical users or just for testing?

Reply:
That's correct - the binary is to illustrate how to use the library, and is not intended to be fully functional like other Sigstore CLI tools (e.g. `cosign`).

Reply:
Do we want to make a note of that in the README?

Reply:
... of course, after I push additional language, I notice right below it we also have:

> Sigstore already has a canonical Go client implementation, [cosign](https://github.com/sigstore/cosign), which was developed with a focus on container image signing/verification. It has a rich CLI and a long legacy of features and development. `sigstore-go` is a more minimal and friendly API for integrating Go code with Sigstore, with a focus on the newly specified data structures in [sigstore/protobuf-specs](https://github.com/sigstore/protobuf-specs).

Reply:
I think your clarification helps - The quote does not explicitly say that sigstore-go as a CLI is discouraged.